frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -1,113 +1,129 @@
+import { WorkspaceMenuTabs } from './components/WorkspaceSideMenu/WorkspaceMenuTabs'
+
+// TODO (hans): Get workspaces data from workspace API once it's implemented
+const MOCK_WORKSPACES_DATA = [
+  {
+    _id: '',
+    title: 'All forms',
+    numForms: 531159249035,
+  },
+  {
+    _id: '2',
+    title: 'Product feedback',
+    numForms: 35002,
+  },
+  {
+    _id: '3',
+    title: 'Public sentiment',
+    numForms: 12,
+  },
+  {
+    _id: '4',
+    title: 'Very long number of forms',
+    numForms: 531159214021,
+  },
+]
+
+const WorkspaceMobilePage = (): JSX.Element => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const [currWorkspaceId, setCurrWorkspaceId] = useState<string>('')
 
-import CreateFormModal from './components/CreateFormModal'
-import { EmptyWorkspace } from './components/EmptyWorkspace'
-import { WorkspaceFormRows } from './components/WorkspaceFormRow'
-import { WorkspaceHeader } from './components/WorkspaceHeader'
-import { useWorkspace } from './queries'
-
-const PAGE_DEFAULTS = {
-  size: 20,
-  pageNumber: 1,
-}
-
-export const CONTAINER_MAXW = '69.5rem'
-
-const useWorkspaceForms = () => {
-  const { data: dashboardForms, isLoading } = useWorkspace()
-  const [searchParams, setSearchParams] = useSearchParams()
-  const [sortedForms, setSortedForms] = useState(dashboardForms)
-  const [isManipulating, setIsManipulating] = useState(false)
-
-  const createFormModalDisclosure = useDisclosure()
-
-  const topRef = useRef<HTMLDivElement>(null)
-
-  const currentPage = Number(
-    searchParams.get('page') ?? PAGE_DEFAULTS.pageNumber,
-  )
-  const pageSize = Number(searchParams.get('size') ?? PAGE_DEFAULTS.size)
-  const sort = searchParams.get('sort')
-
-  const setSortOrder = useCallback(
-    (sort: string) => {
-      setSearchParams({ sort })
-    },
-    [setSearchParams],
-  )
-
-  const setPageNumber = useCallback(
-    (page: number) => {
-      setSearchParams({ page: String(page) })
-    },
-    [setSearchParams],
+  return (
+    <>
+      <Drawer placement="left" onClose={onClose} isOpen={isOpen}>
+        <DrawerOverlay />
+        <DrawerContent maxW="15.5rem">
+          <DrawerHeader p={0}>
+            <Flex pt="1rem" px="1rem" alignItems="center">
+              <IconButton
+                icon={<BiMenuAltLeft />}
+                onClick={onClose}
+                aria-label="close workspace drawer"
+                variant="clear"
+                colorScheme="secondary"
+              />
+              <WorkspaceMenuHeader mt={0} px={0} w="100%" />
+            </Flex>
+          </DrawerHeader>
+          <DrawerBody px={0} pt="1rem">
+            <WorkspaceMenuTabs
+              workspaces={MOCK_WORKSPACES_DATA}
+              currWorkspace={currWorkspaceId}
+              onClick={(id) => {
+                setCurrWorkspaceId(id)
+                onClose()
+              }}
+            />
+          </DrawerBody>
+        </DrawerContent>
+      </Drawer>
```

Comment:
as above, i think only the drawer needs to be its own mobile component. the rest of the workspace page should be able to be styled responsively.

can hmu on slack if you need guidance.

Reply:
Okay thanks, I've updated to use Chakra's `Grid` to do that